### ISSUE TITLE
update citation

### ISSUE THREAD

**Issue by raynamharris** [cienciadedatos/datos#104]
I want to cite this R package, but I noticed that the citation is a little out of data (e.g. old version, missing authors), compared to what is in the DESCRIPTION file. I looked for a CITATION file, but I could not find one. I'm happy to update it if you point me to where the changes need to be made. 

```
> citation("datos")

To cite package ‘datos’ in publications use:

  Edgar Ruiz, Riva Quiroga, Mauricio Vargas and Mauro Lepore (2019). datos: Traduce al
  Español Varios Conjuntos de Datos de Práctica. R package version 0.1.0.
  https://CRAN.R-project.org/package=datos

A BibTeX entry for LaTeX users is

  @Manual{,
    title = {datos: Traduce al Español Varios Conjuntos de Datos de Práctica},
    author = {Edgar Ruiz and Riva Quiroga and Mauricio Vargas and Mauro Lepore},
    year = {2019},
    note = {R package version 0.1.0},
    url = {https://CRAN.R-project.org/package=datos},
  }
```

**Comment by rivaquiroga** [comment]
Hi @raynamharris! The citation is build upon the `DESCRIPTION` file.

**Comment by rivaquiroga** [comment]
I'm closing this issue as it can be solved through #139
